src/components/track-tile/TrackTileRankIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+import { StyleSheet, View } from 'react-native'
+
+import IconCrown from 'app/assets/images/iconCrown.svg'
+import IconTrending from 'app/assets/images/iconTrending.svg'
+import Text from 'app/components/text'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ThemeColors, useThemeColors } from 'app/utils/theme'
+
+import { createStyles as createTrackTileStyles } from './styles'
+
+type Props = {
+  /** Whether or not to show the crown icon */
+  showCrown: boolean
+  /** Index of this item in the lineup */
+  index: number
+  /** Whether or not the rank icon is visible */
+  isVisible?: boolean
+}
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    text: {
+      color: themeColors.secondary
+    },
+    icon: {
+      marginRight: 4
+    }
+  })
+
+const TrackTileRankIcon = ({ showCrown, index, isVisible = true }: Props) => {
```

Comment:
just wondering about passing in a boolean that indicates `return null` instead of controlling that at the parent level?